src/api/routes/edition.js

```diff
@@ -305,8 +305,16 @@ router.get('/',
 	loadEntityRelationshipsForBrowse(),
 	async (req, res, next) => {
 		function relationshipsFilterMethod(relatedEntity) {
-			if (req.query.type) {
-				return _.toLower(_.get(relatedEntity, 'editionType.label')) === req.query.type;
+			const edtionFormatMatched = _.toLower(relatedEntity.editionFormat) === req.query.format;
+			const editionLanguageMatched = relatedEntity.includes(_.upperFirst(req.query.language));
```

Comment:
Here is used the format filter, instead of status. the format is more meaningful than status. Edition formats are paperback, hardcover, ebook extra.